src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -1445,6 +1406,108 @@ def _fill_event_detection_kwargs(
 
         return kwargs
 
+    def _init_columns(
+            self,
+            auto_column_detect: bool = False,
+            trial_columns: str | list[str] | None = None,
+            time_column: str | None = None,
+            time_unit: str | None = None,
+            pixel_columns: list[str] | None = None,
+            position_columns: list[str] | None = None,
+            velocity_columns: list[str] | None = None,
+            acceleration_columns: list[str] | None = None,
+            distance_column: str | None = None,
+    ) -> None:
+        """Initialize dataframe columns."""
```

Comment:
py(doc)lint not complaining about missing docstring definition?

Reply:
not needed for private functions/methods. Here I would just expect some basic knowledge from the reader of the source code, as the columns are already documented for `__init__()`. I would probably document arguments that are not self-explanatory, but I wouldn't care in this instance.